app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="mcs.salazar.jesus.mcsbattleship">
+    <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
Can we avoid declaring the same permissions twice?

Reply:
check the next line

Reply:
I did delete the duplication code.